dotnet/src/webdriver/DevTools/DevToolsSession.cs

```diff
@@ -194,7 +201,7 @@ public async Task<ICommandResponse<TCommand>> SendCommand<TCommand>(TCommand com
         /// <param name="throwExceptionIfResponseNotReceived"><see langword="true"/> to throw an exception if a response is not received; otherwise, <see langword="false"/>.</param>
         /// <returns>The command response object implementing the <see cref="ICommandResponse{T}"/> interface.</returns>
         //[DebuggerStepThrough]
-        public async Task<JToken> SendCommand(string commandName, JToken commandParameters, CancellationToken cancellationToken = default(CancellationToken), int? millisecondsTimeout = null, bool throwExceptionIfResponseNotReceived = true)
+        public async Task<DevToolsCommandResponse> SendCommand(DevToolsCommandSettings devToolsCommandSettings, CancellationToken cancellationToken = default(CancellationToken), int? millisecondsTimeout = null, bool throwExceptionIfResponseNotReceived = true)
```

Comment:
Is this change backwards compatible?

Reply:
Yes, the DevToolsCommandSettings wraps the previous separate (commandName, @params) parameters. This to aline it more with the newly added SendCommand function which accepts a collection of DevToolsCommandSettings.

Reply:
I've re-introduced this public function to be backwards compatible.